app/models/customer.rb

```diff
@@ -1,6 +1,6 @@
 class Customer < ApplicationRecord
-  belongs_to :adress
+  belongs_to :address
   belongs_to :email_marketing_consent
-  belongs_to :metafield
-  belongs_to :sms_marketing_consent
+  # belongs_to :metafield
```

Comment:
Why associations are commented?

Reply:
I don't see them in the response when I request customers. I'll try to check what else can be done, but for now I left it like that.